src/bridge/MatrixHandler.ts

```diff
@@ -964,8 +964,13 @@ export class MatrixHandler {
                 cacheBody = reply.reply;
             }
         }
+        let body = cacheBody.trim().substr(0, REPLY_SOURCE_MAX_LENGTH);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr is technically deprecated in favour of substring.